src/rust/engine/src/nodes.rs

```diff
@@ -1020,17 +1020,21 @@ impl Node for NodeKey {
 
     let (started_workunit_id, user_facing_name) = {
       let user_facing_name = self.user_facing_name();
-      let display = context.session.should_handle_workunits()
+      let higher_priority = context.session.should_handle_workunits()
         && (user_facing_name.is_some() || self.display_info().is_some());
+      let level = if higher_priority {
+        Level::Info
+      } else {
+        Level::Trace
```

Comment:
Although we'll have to figure out conventions as we go: as a rule of thumb, my guess is that no `Node` implementation should be below `Debug`. Nodes are relatively heavyweight.

Things like "uploading individual digests to the network" might be "Trace" level workunits.